middleware/timeout.go

```diff
@@ -15,16 +19,17 @@ type (
 		// Skipper defines a function to skip middleware.
 		Skipper Skipper
 
-		// ErrorMessage is written to response on timeout in addition to http.StatusServiceUnavailable (503) status code
```

Comment:
This `ErrorMessage` cannot be removed just like that. This would break the promise of backwards compatibility. However, I agree we can ignore it. Maybe a deprecation mark in the docs would do just fine here. Another option would be to use it in the default `DefaultTimeoutErrorHandler` but here I can't think of anything helpful to be honest.